app/MMB/src/scenes/help/Help.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import { ScrollView, Platform, Linking } from 'react-native';
+import { TextIcon, StyleSheet, Color } from '@kiwicom/mobile-shared';
+import { Translation, Alert } from '@kiwicom/mobile-localization';
+import {
+  MenuItem,
+  MenuGroup,
+  SeparatorFullWidth,
+  SeparatorTrimmed,
+  type RouteNamesType,
+  type NavigationType,
+} from '@kiwicom/mobile-navigation';
+
+import type { HelpContainerQueryResponse } from './__generated__/HelpContainerQuery.graphql';
+
+type Props = {|
+  +navigation: NavigationType,
+  +data: HelpContainerQueryResponse,
+|};
+
+export default class Help extends React.Component<Props> {
+  navigate = (key: RouteNamesType) => {
+    this.props.navigation.navigate(key);
+  };
+
+  handleOpenHelp = () => {
+    this.navigate('mmb.help.help');
+  };
+
+  handleOpenCallSupport = () => {
```

Comment:
Please try to refactor this function a bit, to many levels of if/else and please also try to use async/await

Reply:
I'm on it.